<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>main_window</class>
 <widget class="QWidget" name="main_window">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>661</width>
    <height>536</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Calculator</string>
  </property>
  <widget class="QPushButton" name="number_e_b">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>460</y>
     <width>81</width>
     <height>71</height>
    </rect>
   </property>
   <property name="text">
    <string>e</string>
   </property>
  </widget>
  <widget class="QPushButton" name="number_0_b">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>460</y>
     <width>81</width>
     <height>71</height>
    </rect>
   </property>
   <property name="text">
    <string>0</string>
   </property>
  </widget>
  <widget class="QPushButton" name="number_pi_b">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>460</y>
     <width>81</width>
     <height>71</height>
    </rect>
   </property>
   <property name="text">
    <string>π</string>
   </property>
  </widget>
  <widget class="QPushButton" name="brace_left">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>460</y>
     <width>81</width>
     <height>71</height>
    </rect>
   </property>
   <property name="text">
    <string>(</string>
   </property>
  </widget>
  <widget class="QPushButton" name="brace_right">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>460</y>
     <width>81</width>
     <height>71</height>
    </rect>
   </property>
   <property name="text">
    <string>)</string>
   </property>
  </widget>
  <widget class="QPushButton" name="function_log">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>460</y>
     <width>81</width>
     <height>71</height>
    </rect>
   </property>
   <property name="text">
    <string>log()</string>
   </property>
  </widget>
  <widget class="QPushButton" name="result">
   <property name="geometry">
    <rect>
     <x>550</x>
     <y>460</y>
     <width>100</width>
     <height>71</height>
    </rect>
   </property>
   <property name="text">
    <string>=</string>
   </property>
  </widget>
  <widget class="QPushButton" name="number_1_b">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>380</y>
     <width>80</width>
     <height>71</height>
    </rect>
   </property>
   <property name="text">
    <string>1</string>
   </property>
  </widget>
  <widget class="QPushButton" name="number_2_b">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>380</y>
     <width>80</width>
     <height>71</height>
    </rect>
   </property>
   <property name="text">
    <string>2</string>
   </property>
  </widget>
  <widget class="QPushButton" name="number_3_b">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>380</y>
     <width>80</width>
     <height>71</height>
    </rect>
   </property>
   <property name="text">
    <string>3</string>
   </property>
  </widget>
  <widget class="QPushButton" name="operation_exp">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>380</y>
     <width>80</width>
     <height>71</height>
    </rect>
   </property>
   <property name="text">
    <string>xʸ</string>
   </property>
  </widget>
  <widget class="QPushButton" name="brace_abs">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>380</y>
     <width>80</width>
     <height>71</height>
    </rect>
   </property>
   <property name="text">
    <string>| x |</string>
   </property>
  </widget>
  <widget class="QPushButton" name="function_tg">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>380</y>
     <width>80</width>
     <height>71</height>
    </rect>
   </property>
   <property name="text">
    <string>tg()</string>
   </property>
  </widget>
  <widget class="QPushButton" name="ans_b">
   <property name="geometry">
    <rect>
     <x>550</x>
     <y>380</y>
     <width>100</width>
     <height>71</height>
    </rect>
   </property>
   <property name="text">
    <string>Ans</string>
   </property>
  </widget>
  <widget class="QPushButton" name="function_sin">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>220</y>
     <width>80</width>
     <height>71</height>
    </rect>
   </property>
   <property name="text">
    <string>sin()</string>
   </property>
  </widget>
  <widget class="QPushButton" name="number_5_b">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>300</y>
     <width>81</width>
     <height>71</height>
    </rect>
   </property>
   <property name="text">
    <string>5</string>
   </property>
  </widget>
  <widget class="QPushButton" name="operation_devide">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>300</y>
     <width>81</width>
     <height>71</height>
    </rect>
   </property>
   <property name="text">
    <string>/</string>
   </property>
  </widget>
  <widget class="QPushButton" name="number_8_b">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>220</y>
     <width>80</width>
     <height>71</height>
    </rect>
   </property>
   <property name="text">
    <string>8</string>
   </property>
  </widget>
  <widget class="QPushButton" name="delete_all">
   <property name="geometry">
    <rect>
     <x>550</x>
     <y>220</y>
     <width>100</width>
     <height>71</height>
    </rect>
   </property>
   <property name="text">
    <string>C</string>
   </property>
  </widget>
  <widget class="QPushButton" name="number_6_b">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>300</y>
     <width>81</width>
     <height>71</height>
    </rect>
   </property>
   <property name="text">
    <string>6</string>
   </property>
  </widget>
  <widget class="QPushButton" name="number_9_b">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>220</y>
     <width>80</width>
     <height>71</height>
    </rect>
   </property>
   <property name="text">
    <string>9</string>
   </property>
  </widget>
  <widget class="QPushButton" name="operation_multiply">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>300</y>
     <width>81</width>
     <height>71</height>
    </rect>
   </property>
   <property name="text">
    <string>×</string>
   </property>
  </widget>
  <widget class="QPushButton" name="delete_symbol">
   <property name="geometry">
    <rect>
     <x>550</x>
     <y>300</y>
     <width>100</width>
     <height>71</height>
    </rect>
   </property>
   <property name="text">
    <string>Del</string>
   </property>
  </widget>
  <widget class="QPushButton" name="function_cos">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>300</y>
     <width>81</width>
     <height>71</height>
    </rect>
   </property>
   <property name="text">
    <string>cos()</string>
   </property>
  </widget>
  <widget class="QPushButton" name="number_7_b">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>220</y>
     <width>80</width>
     <height>71</height>
    </rect>
   </property>
   <property name="text">
    <string>7</string>
   </property>
  </widget>
  <widget class="QPushButton" name="operation_plus">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>220</y>
     <width>80</width>
     <height>71</height>
    </rect>
   </property>
   <property name="text">
    <string>+</string>
   </property>
  </widget>
  <widget class="QPushButton" name="number_4_b">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>300</y>
     <width>81</width>
     <height>71</height>
    </rect>
   </property>
   <property name="text">
    <string>4</string>
   </property>
  </widget>
  <widget class="QPushButton" name="operation_minus">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>220</y>
     <width>80</width>
     <height>71</height>
    </rect>
   </property>
   <property name="text">
    <string>-</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="input_order">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>140</y>
     <width>641</width>
     <height>70</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="last_operation">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>20</y>
     <width>641</width>
     <height>111</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
